commands/operator-sdk/cmd/scorecard/resource_handler.go

```diff
@@ -168,6 +143,41 @@ func createFromYAMLFile(yamlPath string) error {
 	return nil
 }
 
+// getProxyPodFromDeployment returns a deployment depName's pod in namespace.
+func getProxyPodFromDeployment(depName, namespace string) (pod *v1.Pod, err error) {
```

Comment:
This util is only really fetching the pod for a Deployment. Nothing specific about the proxy. In fact it's just the operator pod with the proxy container running alongside the operator.

So probably rename it to make it clearer:
```Go
// getPodForDeployment returns the singular pod for the given deployment name and namespace.
func getPodForDeployment(depName, namespace string) (pod *v1.Pod, err error) {
```